cmd/kyma/alpha/create/module/module.go

```diff
@@ -158,6 +158,13 @@ func (cmd *command) Run(args []string) error {
 
 	cmd.CurrentStep.Successf("Image created")
 
+	/* -- ADD SECURITY SCANNING METADATA -- */
+	cmd.NewStep("Configuring security scanning...")
```

Comment:
can we change this so that the step is only activated if the file is found? that way you can silently opt out of this process if the file is not present. Alternative would be to define a bool flag which is disabled by default and can take in a file